judgments/fixtures/stop_words.py

```diff
@@ -135,13 +133,9 @@
     "should",
     "should've",
     "now",
-    "d",
     "ll",
```

Comment:
Is it worth reviewing this whole list for things like this, which only exist as part of contractions?